### DISCUSSION TITLE
Backup & Restore config results in bricked ESP8266

### DISCUSSION THREAD

**Discussion by schweini** [arendst/Tasmota#21952] [Q&A]
I tried to 'clone' the config of WeMos Devkit running the latest release of tasmota-sensors to another, identical board by makig a .dmp backup of the configuration, and restoring it to the other device. This resultas in a board that is completely bricked (no traffic on the serial monitor) and has to be re-flashed. 

I even tried doing a backup and restore on the same board, and it has the same effect.

This is on Tasmota 14.1.0 (release-sensors)

**Comment by sfromis** [comment]
I just did a small test, with tasmota-sensors on a NodeMCU. Did a save configuration, and then restoring the saved configuration to the same device. It booted normally, without any trouble.
